.github/CODEOWNERS

```diff
@@ -550,4 +550,10 @@
 /package.json @vojtechjelinek @kevinlee12
 /yarn.lock @vojtechjelinek @kevinlee12
 /scripts/install_third_party_libs.py @vojtechjelinek @kevinlee12
-/.github/ @kevinlee12
+/.github/* @kevinlee12
+
+# Procedure for picking code owners:
+# Codeowners should be selected based on speciality from the following list:
+# https://github.com/oppia/oppia/projects
```

Comment:
I'm not sure whether this comment is required here. [Maybe you can add this on top?]

Reply:
Added to the top.